scriptmodules/supplementary/skyscraper.sh

```diff
@@ -658,3 +610,195 @@ function gui_skyscraper() {
         fi
     done
 }
+
+function _gui_cache_skyscraper() {
+    local db_size
+    local cache_folder=$(_cache_folder_skyscraper)
+    local -A help_strings_cache
```

Comment:
for associate arrays we tend to use declare over local. Better documented in bash.